packages/devtools_app/test/cpu_profile_model_test.dart

```diff
@@ -2,11 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: avoid_redundant_argument_values
+// ignore_for_file: avoid_redundant_argument_values, import_of_legacy_library_into_null_safe
 
-import 'package:devtools_app/src/primitives/utils.dart';
-import 'package:devtools_app/src/screens/profiler/cpu_profile_model.dart';
+import 'package:devtools_app/devtools_app.dart';
```

Comment:
do not import this file. instead import the exact file you need. Your IDE may auto suggest you import this file (this file exports a bunch of other files in devtools), but you'll always want to make sure you are using the exact library you need instead of this catch-all lib.